### PR TITLE
fix: always consider timezone when set allday schedule period(fix #662)

### LINKED ISSUE #662: Schedule.prototype.setAllDayPeriod  logic
## Summary
```js
Schedule.prototype.setAllDayPeriod = function(start, end) {
    // If it is an all-day schedule, only the date information of the string is used.
    if (util.isString(start)) {
        start = datetime.parse(start.substring(0, 10));
    } else {
        start = new TZDate(start || Date.now());
    }
    if (util.isString(end)) {
        end = datetime.parse(end.substring(0, 10));
    } else {
        end = new TZDate(end || this.start);
    }

    this.start = start;
    this.start.setHours(0, 0, 0);
    this.end = end || new TZDate(this.start);
    this.end.setHours(23, 59, 59);
};
```
I don't understand why to use ```if (util.isString(start)) {
        start = datetime.parse(start.substring(0, 10));
    } ``` to deal with string arg as a special case, because```new TZDate()```accept string arg.

in my case, I store start-date value as json date, for example ''2020-07-06T16:00:00.000Z'' (my timezone is +8)
 ```datetime.parse(start.substring(0, 10))```
will give me 2020-07-06 as the start-date, which is not what I want.
```new TZDate(start)```give me the right date which is 2020-07-07.

As summary,  ```datetime.parse(start.substring(0, 10))``` will not consider **timezone** like ```new TZDate(start)``` do, which lead to logical inconsistency.

## Version
1.12.13

### PR DESCRIPTION
**Pull request by lhysdl**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

**Comment by lhysdl** [comment]
@jungeun-cho

**Comment by jungeun-cho** [comment]
ok to test

File: src/js/model/schedule.js [hunk 1]
```diff
@@ -269,21 +269,9 @@ Schedule.prototype.init = function(options) {
...
-    } else {
-        end = new TZDate(end || this.start);
-    }
-
-    this.start = start;
+    this.start = new TZDate(start || Date.now());
     this.start.setHours(0, 0, 0);
-    this.end = end || new TZDate(this.start);
+    this.end = new TZDate(end || this.start);
     this.end.setHours(23, 59, 59);
```
Comment (jungeun-cho) [inline src/js/model/schedule.js:275]: You can use `start`, `end` of datetime
https://github.com/nhn/tui.calendar/blob/master/src/js/common/datetime.js#L396
https://github.com/nhn/tui.calendar/blob/master/src/js/common/datetime.js#L408
```js
    this.start = datetime.start(new TZDate(start || Date.now()));
    this.end = datetime.end(new TZDate(end || this.start));
```

**Comment by jungeun-cho** [comment]
@dongsik-yoo check this PR.
I will modify the IE9 related TC. (parse date format issues +09:00)

File: src/js/model/schedule.js (hunk 1 above)
Comment (lhysdl) [reply src/js/model/schedule.js]: just modified.

**Review by jungeun-cho — APPROVED** [review]

**Pull request merged**